recipe/meta.yaml

```diff
@@ -9,8 +9,8 @@ source:
   sha256: 806e034dda44114815e23c16ef92f95c91e4c71100ff52813adf7132a6ad870d
 
 build:
-  number: 1
-  noarch: python
+  number: 2
+  skip: true  # [py<37]
```

Comment:
If we make this version as an arch-specific package then different architects will be available for `conda search/install` to pick up. And that's not good. So do not skip if you rebuilding the package and bumping the build number